terraform/bosh/rds.tf

```diff
@@ -0,0 +1,61 @@
+resource "aws_db_subnet_group" "bosh_rds" {
+  name = "${var.env}-bosh"
+  description = "Subnet group for BOSH RDS"
+  subnet_ids = [ "${split(",", var.infra_subnet_ids)}"  ]
+
+  tags {
+      Name = "${var.env}-bosh-rds"
+  }
+}
+
+resource "aws_security_group" "bosh_rds" {
+  name = "${var.env}-bosh-rds"
+  description = "BOSH RDS security group"
+  vpc_id = "${var.vpc_id}"
+
+  ingress {
+    from_port = 5432
+    to_port   = 5432
+    protocol  = "tcp"
+    security_groups = [
+      "${aws_security_group.bosh.id}",
+      "${var.concourse_security_group_id}",
+    ]
+  }
+
+  tags {
+    Name = "${var.env}-bosh-rds"
+  }
+}
+
+resource "aws_db_parameter_group" "default" {
+    name = "${var.env}-bosh"
+    family = "postgres9.4"
+    description = "RDS Postgres default parameter group"
+}
+
+resource "aws_db_instance" "bosh" {
+  identifier = "${var.env}-bosh"
+  name = "bosh"
+  allocated_storage = 10
```

Comment:
Utilisation of this DB seems to be around 340MB, so is 10GB allocation perhaps a bit overkill here - we could save a few pennies off our hosting bill with a smaller value.